### PR TITLE
YSQL Regression Tests for Aggregate Functions

### PR DESCRIPTION
**Pull request by ngov17**
This pull request adds regression tests for Aggregate functions from the postgres framework `(src/postgres/src/test/regress/)`  which include `yb_pg_aggregates` and `yb_aggregates`. 

In light of this new schedule for aggregate functions, the test for` yb_pg_aggregates` in` yb_pg_misc_serial_schedule` has been deleted in pull request https://github.com/yugabyte/yugabyte-db/pull/5264.

This pull request was originally part of the larger pull request https://github.com/yugabyte/yugabyte-db/pull/5190 (now edited) which has now been broken down into 4 separate pull requests:  https://github.com/yugabyte/yugabyte-db/pull/5190 (edited), https://github.com/yugabyte/yugabyte-db/pull/5262, https://github.com/yugabyte/yugabyte-db/pull/5263, https://github.com/yugabyte/yugabyte-db/pull/5264.

### CODE REVIEW

**Comment by jaki** [comment]
> In light of this new schedule for aggregate functions, the test for `yb_pg_aggregates` in `yb_pg_misc_serial_schedule` has been deleted in pull request #5264.

Can this change be moved to this pull request?

**Comment by jaki** [comment]
Also take `yb_aggregates` test out of `yb_misc_serial_schedule`.  You could have checked for this using `git grep yb_aggregates` from `src/postgres/src/test/regress`, for example.

**Comment by ngov17** [comment]
@jaki I've made the changes

File: src/postgres/src/test/regress/yb_pg_misc_serial_schedule [hunk 1]
```diff
@@ -28,4 +24,3 @@ test: yb_pg_create_aggregate
```
Comment (jaki) [inline src/postgres/src/test/regress/yb_pg_misc_serial_schedule:24]: Please remove `yb_pg_create_aggregate` from here.  Also check whether the above dependencies are still needed now that this is gone.

**Review by jaki — CHANGES_REQUESTED** [review]
Just caught this.

**Comment by ngov17** [comment]
@jaki I've made the changes. Thanks!

File: src/postgres/src/test/regress/yb_pg_misc_serial_schedule (hunk 1 above)
Comment (ngov17) [reply src/postgres/src/test/regress/yb_pg_misc_serial_schedule]: The dependencies will still be needed for the other tests.

**Pull request merged**